homeassistant/components/overkiz/lock.py

```diff
@@ -44,7 +44,6 @@ async def async_unlock(self, **kwargs: Any) -> None:
     @property
     def is_locked(self) -> bool | None:
         """Return a boolean for the state of the lock."""
-        return (
-            self.executor.select_state(OverkizState.CORE_LOCKED_UNLOCKED)
-            == OverkizCommandParam.LOCKED
+        return self.executor.select_state(OverkizState.CORE_LOCKED_UNLOCKED) == cast(
```

Comment:
The cast should not be needed, as this isn't a comparison by id; but a loose comparison